### PR TITLE
V1.7

### PR DESCRIPTION
**Pull request by lheisler**
moved to a new version of the nextflow workflow, installed in modulator
changes in wdl account for changes in the nextflow output

### CODE REVIEW

File: ncov2019ArticNf.wdl [hunk 1]
```diff
@@ -30,8 +30,8 @@ workflow ncov2019ArticNf {
     File trimPrimerSequencesBam = illumina_ncov2019ArticNf.trimPrimerSequencesBam
     File trimPrimerSequencesPrimerTrimmedBam = illumina_ncov2019ArticNf.trimPrimerSequencesPrimerTrimmedBam
     File makeConsensusFasta = illumina_ncov2019ArticNf.makeConsensusFasta
+	File makeConsensusVcf = illumina_ncov2019ArticNf.makeConsensusVcf
```
Comment (apmasell) [inline ncov2019ArticNf.wdl]: Weird indentation on this line. I think the file is indented with spaces.

**Review by apmasell — APPROVED** [review]

**Comment by lheisler** [comment]
retest this please

File: ncov2019ArticNf.wdl (hunk 1 above)
Comment (lheisler) [reply ncov2019ArticNf.wdl]: fixed that

**Review by xuemeiluo — APPROVED** [review]

**Comment by lheisler** [comment]
retest this please

**Pull request merged**